src/control/SConscript

```diff
@@ -44,14 +44,14 @@ def is_ucx_build(benv):
 
 def get_build_tags(benv):
     "Get custom go build tags."
-    tags = ""
+    tags = []
     if is_firmware_mgmt_build(benv):
-        tags += "firmware "
+        tags.append("firmware")
     if is_ucx_build(benv):
-        tags += "ucx "
+        tags.append("ucx")
     if tags == "":
```

Comment:
Oops, this test will always be false. I think you want the following:
```suggestion
    if len(tags) == 0:
```

Reply:
Oops!